src/index.html

```diff
@@ -35,24 +36,24 @@
           <div class="stars__star"></div>
         </div>
 
-        <article class="reviews_amount">
+        <article class="product__reviews_amount">
```

Comment:
let's rname elements according to[ BEM naming convention](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem) we use here pls check everywhere